presto-main/src/main/java/com/facebook/presto/operator/aggregation/builder/InMemoryHashAggregationBuilder.java

```diff
@@ -118,6 +118,12 @@ public InMemoryHashAggregationBuilder(
     @Override
     public void close()
     {
+        if (partial) {
```

Comment:
updateMemory does not take parameters and calls getSizeInMemory. We can do it that way, but then we need to set all reference to null so getSizeInMemory would return 0. If we do that all code need to check for null pointers. There's pro & con for both. This way is a smaller code change.